.github/workflows/workflow_pr.yml

```
@@ -25,6 +25,10 @@ defaults:
     shell: bash
 
 jobs:
+  test:
+    name: Jest Tests
+    uses: ./.github/workflows/test_job.yml
```

I think this needs a `working_directory` setup here in both workflows too
```suggestion
    working_directory: ./service-add-lpa-to-pod
    uses: ./.github/workflows/test_job.yml
```

---

Yep, just added and pushed that, will take a look at the docker change above